.ci_support/environment-notebooks-damask.yml

```diff
@@ -0,0 +1,21 @@
+channels:
+  - conda-forge
+dependencies:
+  - ase =3.22.1
+  - coveralls
+  - coverage
+  - codacy-coverage
+  - damask-mesh =3.0.0a8
+  - damask-grid =3.0.0a8
+  - damask =3.0.0a8
+  - matplotlib =3.8.0
+  - numpy =1.26.3
+  - pyiron =0.5.0
+  - pyiron-data =0.0.26
+  - python-damask =3.0.0a8
+  - pyvista =0.42.3
+  - scipy =1.11.4
+  - seaborn =0.13.1
+  - scikit-image =0.22.0
+  - papermill
+  - jupyter
```

Comment:
Since most of the damask and fenics envs are shared, I would suggest showing this explicitly with `environment-notebooks-shared/damask/fenics.yml` files, and then merge shared+damask and shared+fenics. This will reduce the overhead of keeping all the shared versions sync'd. Some of this might even move into the core (non-notebook) env, such that the tests use merges of core+shared+damask, core+shared+fenics

Reply:
Check [this centralized action](https://github.com/pyiron/actions/blob/main/write-environment/action.yml) for a tool to throw in the CI scripts for merging envs like this